### DISCUSSION TITLE
[Idea] Use org build and release workflow

### DISCUSSION THREAD

**Discussion by leemyongpakvn** [PrestaShop/PrestaShop#32901] [Ideas]
### The problem

Current build-release.yml of some native modules are outdated, that caused warnings in recent PRs such as [autoupgrade Release 4.15.0](https://github.com/PrestaShop/autoupgrade/actions/runs/3739690872) or [ps_mainmenu Release 2.3.2](https://github.com/PrestaShop/ps_mainmenu/actions/runs/4056715254). Using [organization build and release workflow](https://github.com/PrestaShop/.github/blob/master/.github/workflows/build-release.yml) will solve it.
In fact, @atomiix created organization build and release workflow and used it for some native modules' build-release.yml in 2022. But he becomes less active this year, so I create this issue to track the progress of updating build-release.yml for the remaining native modules.

- [ ] autoupgrade
- [x] blockreassurance
- [x] blockwishlist
- [ ] contactform
- [ ] dashactivity
- [ ] dashgoals
- [ ] dashproducts
- [ ] dashtrends
- [ ] graphnvd3
- [ ] gridhtml
- [ ] gsitemap
- [x] pagesnotfound
- [x] productcomments
- [ ] ps_banner
- [ ] ps_bestsellers
- [ ] ps_brandlist
- [ ] ps_cashondelivery
- [ ] ps_categoryproducts
- [ ] ps_categorytree.yml
- [ ] ps_checkpayment
- [ ] ps_contactinfo
- [x] ps_crossselling
- [ ] ps_currencyselector
- [x] ps_customeraccountlinks
- [x] ps_customersignin
- [ ] ps_customtext
- [x] ps_dataprivacy
- [x] ps_distributionapiclient
- [x] ps_emailalerts
- [x] ps_emailsubscription
- [x] ps_facetedsearch
- [ ] ps_faviconnotificationbo
- [ ] ps_featuredproducts
- [x] ps_googleanalytics
- [x] ps_imageslider 
- [x] ps_languageselector
- [x] ps_linklist
- [ ] ps_mainmenu
- [x] ps_newproducts
- [x] ps_searchbar.yml
- [x] ps_sharebuttons
- [ ] ps_shoppingcart
- [x] ps_socialfollow
- [x] ps_specials
- [x] ps_supplierlist
- [x] ps_themecusto
- [x] ps_viewedproduct
- [ ] ps_wirepayment
- [x] psgdpr
- [ ] statsbestcategories
- [x] statsbestcustomers
- [x] statsbestmanufacturers
- [ ] statsbestproducts
- [x] statsbestsuppliers
- [ ] statsbestvouchers
- [ ] statscarrier
- [x] statscatalog
- [x] statscheckup
- [x] statsdata
- [ ] statsforecast
- [ ] statsnewsletter
- [ ] statspersonalinfos
- [x] statsproduct
- [x] statsregistrations
- [ ] statssales
- [ ] statssearch
- [x] statsstock

### The solution

Just copy the updated build-release.yml content and override the outdated one content to complete the tasklist.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by matks** [comment]
I wanted to _transform_ this Discussion into a GitHub Issue (= Feature Request) but it created a new issue instead of transforming this one :/ https://github.com/PrestaShop/PrestaShop/issues/32902

**Comment by kpodemski** [comment]
Hi @leemyongpakvn 

What about keeping [this issue](https://github.com/PrestaShop/PrestaShop/issues/32902) as a center of truth for this project?

**Comment by leemyongpakvn** [comment]
@kpodemski Yes, I'm using it to track the progress of Build-Release update for all native modules. Open or Close status of it doesn't affect me :)

**Comment by kpodemski** [comment]
@leemyongpakvn In that case I'm closing this Discussion and let's continue in the Issue :-)
